### ISSUE TITLE
Element wise MeanSquaredError

### ISSUE THREAD

**Issue by Dankomaister** [atomistic-machine-learning/schnetpack#47]
Hi,

I'm getting an error when i try and run `RootMeanSquaredError('forces','dydx',element_wise=True)` in my training script.

I fixed the problem by changing
`self.n_entries += torch.sum(batch[Structure.atom_mask]) * y.shape[-1]`
in line 135 in metrics.py to
`self.n_entries += torch.sum(batch[Structure.atom_mask]).detach().cpu().data.numpy() * y.shape[-1]`
is this a good solution?

/Daniel

**Comment by Stefaanhess** [comment]
Hey Daniel,

I added your solution in #48 .

Stefaan

**Comment by Dankomaister** [comment]
Thx!

/Daniel
